pkg/skaffold/deploy/resources/resource.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/color"
+)
+
+const (
+	TabHeader = " -"
+)
+
+type ResourceObj struct {
+	name      string
+	namespace string
+	rType     string
+	status    Status
+}
+
+func (r *ResourceObj) String() string {
```

Comment:
I think namespace would be interesting here too especially in multinamespace deployments